acceptance/vars/rec-dev.py

```diff
@@ -0,0 +1,14 @@
+import os
+
+BROWSER = "headlesschrome"
+GK_URL = "https://rec.geokrety.org/"
+GK_URL_RUCHY = GK_URL + "ruchy.php"
+
+WELCOME_NEWS_EXTRACT = "Enjoying GeoKrety.org"
+WELCOME_NEWS_DATE_USER = "2018-04-17 (GK Team)"

```

Comment:
This may not work on a long term if we synchronize database from prod in a few month. But I think we are aware of it ;)

Reply:
sure
will try to minimize ad much as possible 'data related' values of course. 

Maybe need to explore how to build an extension to extract such things from an api call.

Anyway robotfwk steps rely on portal behavior/code update so would be great to keep in mind to minimize future changes or have dedicated config file.

Stay a lots of questions around this kind of tests :)

Reply:
I finally decided to remove data related content